scrape-urls.sh

```diff
@@ -79,8 +79,24 @@ pushd 2023-TK
 
     # F
 
+    # G
+    ../urls-from-html.py https://www.geertruidenberg.nl/uitslagen-van-de-verkiezingen ".pdf" https://www.geertruidenberg.nl > "$(stad 0779)"
+    ../urls-from-html.py https://www.geldrop-mierlo.nl/voorlopige-uitslag-tweede-kamerverkiezingen-2023 > "$(stad 1771)"
+    ../urls-from-html.py https://www.gemert-bakel.nl/uitslagen-verkiezingen "pdf" https://www.gemert-bakel.nl > "$(stad 1652)"
+    ../urls-from-html.py https://www.gennep.nl/uitslag-tweede-kamerverkiezing-2023 > "$(stad 0907)"
+    ../urls-from-html.py https://www.gilzerijen.nl/verkiezingsuitslagen ".pdf" https://www.gilzerijen.nl > "$(stad 0784)"
+    ../urls-from-html.py https://www.goeree-overflakkee.nl/uitslag-tweede-kamerverkiezing-2023 | grep -v docreader > "$(stad 1924)"
+    ../urls-from-html.py https://www.goes.nl/definitieve-uitslag-tweede-kamerverkiezing-2023 > "$(stad 0664)"
+    ../urls-from-html.py https://www.goirle.nl/verkiezingen ".pdf" https://www.goirle.nl > "$(stad 0785)"
+    ../urls-from-html.py https://gooisemeren.nl/verkiezingen-tweede-kamer-22-november-2023/uitslag-tweede-kamerverkiezing-22-november-2023/ ".pdf" https://gooisemeren.nl > "$(stad 1942)"
+    ../urls-from-html.py https://www.gorinchem.nl/tweede-kamerverkiezingen-2023/processen-verbaal-tweede-kamerverkiezingen-2023 > "$(stad 0512)"
+    ../urls-from-html.py https://www.gouda.nl/bestuur-en-organisatie/tweede-kamerverkiezingen/processen-verbaal-tweede-kamerverkiezing/ > "$(stad 0513)"
+    wget -nv -nd -nc -np -r --remote-encoding=utf-8 --local-encoding=utf-8 --spider --no-parent https://documenten.groningen.nl/TK2023/ 2>&1 | grep "pdf 200 OK" | awk '{print $4}' > "$(stad 0014)"
```

Comment:
Dit werkt bij mij niet op macOS, maar de gevonden URL's werken, dus laat maar even zo. Ik denk dat dit scrape script in Python herschreven moet worden om dat soort issues te voorkomen, maar dat kan later wel.